### ISSUE TITLE
Error: Unable to process command '##[set-env name=GIT_TAG_NAME;]stage-v1' successfully.

### ISSUE THREAD

**Issue by jpainam** [olegtarasov/get-tag#296]
Using
```
 - name: Get tag name
        uses: olegtarasov/get-tag@v2
```
fails with 

```
Error: Unable to process command '##[set-env name=GIT_TAG_NAME;]stage-v1' successfully.
Error: The `set-env` command is disabled. Please upgrade to using Environment Files or opt into unsecure command execution by setting the `ACTIONS_ALLOW_UNSECURE_COMMANDS` environment variable to `true`. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

**Comment by brechtm** [comment]
Duplicate of #14. Fixed by using `olegtarasov/get-tag@v2.1`.

**Comment by jpainam** [comment]
Thanks
